src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -234,6 +234,11 @@ public string ProjectVersion
         /// </summary>
         public bool BreakOnInitialTestFailure { get; set; }
 
+        /// <summary>
+        /// The port used when realtime reporting is enabled (`-o` flag).
+        /// </summary>
+        public int? Port { get; init; }
```

Comment:
It this necessary as an option? Can we not just request any free high-range port?

Reply:
Yeah I was already looking at this:
```csharp
static int FreeTcpPort()
{
  TcpListener l = new TcpListener(IPAddress.Loopback, 0);
  l.Start();
  int port = ((IPEndPoint)l.LocalEndpoint).Port;
  l.Stop();
  return port;
}
```

I suppose I should use this instead

Reply:
I also found that stackoverflow suggestion but as far as I can tell what you would be doing is request a port, then release the port, then hope the port is still available by the time you create your new listener.

I'm not sure if all that is necessary, I think you could use 0 for the port number in your current implementation, then check on the listener which port it received and getting that injected into the report.